source/plugin/modularization.rst

```diff
@@ -0,0 +1,61 @@
+===================
+Code Modularization
+===================
+
+When writing Sponge plugins, you may find that you need to modularize your code. Maybe you want to split your core into
+another project so that others can make use of it, or maybe you want to utilize a service another plugin provides. With
+Sponge, begone the days of shading or shadowing entire plugins into your plugin.
```

Comment:
"gone are" would be more appropriate than "begone". Also, there's no need for "or shadowing" since shading and shadowing mean the same thing.